src/Platform/Microsoft.Testing.Platform/Resources/PlatformResources.resx

```diff
@@ -671,4 +671,8 @@ Takes one argument as string in the format &lt;value&gt;[h|m|s] where 'value' is
   <data name="ConfigurationFileNotFound" xml:space="preserve">
     <value>The configuration file '{0}' specified with '--config-file' could not be found.</value>
   </data>
-</root>
+  <data name="TimeoutInRegexStackTraceLineParsing" xml:space="preserve">
+    <value>Parsing stack trace line with regex timeout: '{0}'</value>
```

Comment:
Feels grammatically more correct this way?

```suggestion
    <value>Parsing stack trace line '{0}' with regex timed out.</value>
```

Reply:
yes, but then the long text you are matching will be in the middle of the sentence, and you will never learn what the problem is until you read till the end. 

In the original way you see: <this is problem>: <this is additional info for that problem.>